### ISSUE TITLE
Life Oracle/Scale Fist AC stacking

### ISSUE THREAD

**Issue by JNShin418** [Vek17/WrathMods-TabletopTweaks#203]
**Is your feature request related to a problem? Please describe.**
I have a run that's utilizing the double stacking, and I thought this was a feature since they're different AC categories..

**Describe the solution you'd like**
 Could you make something like a toggle option to allow this?

**Describe alternatives you've considered**

**Additional context**

**Comment by Amariithynar** [comment]
> **Is your feature request related to a problem? Please describe.** I have a run that's utilizing the double stacking, and I thought this was a feature since they're different AC categories..

This is correct. Scaled Fist makes Wisdom-scaling effects instead use Charisma. The Charisma to AC effect of the Scaled Monk is the Monk's ability AC Bonus (Ex); that adds the Wisdom Bonus as a bonus to AC and CMD as well as adds a separate +1 for every 4 levels of Monk so long as you are not wearing armor and are unencumbered. The Nature Oracle's effect is substituting Cha for Dex. By RAW, while using the FAQ against stat stacking, they do stack, as only the Nature Oracle's effect is a stat-typed modifier, which would not stack with itself.

> There is already an option to disable this in the settings

I went looking in the user settings JSON, and was not able to find it, myself... Tried looking for armor-related aspects, same with anything that featured 'scaling', 'stacking', 'AC', 'armor', 'scaled', 'nature', 'CHA', 'bonus', and more as possible terminology, and wasn't able to find it amongst any that would show in list if there were any (the last two I noted don't have any, as far as I know). I assume I'm just passing by it and not grokking that it's the relevant entry. What's the specific terminology used for this change?

**Comment by dtndmd** [comment]
NaturesWhisperMonkStacking
It could be this. I hope there is more explanation of fixes.json, many changes do not know what it means

**Comment by Amariithynar** [comment]
> NaturesWhisperMonkStacking
Okay, that's really weird, because that doesn't exist at all in my fixes.json. I wonder if this has to do with me updating the mod with latest releases and not fully uninstalling them and installing the new one?

**Comment by Vek17** [comment]
It will only exist if you run the game with the new version installed. Settings do not update until after the game is launched

**Comment by Amariithynar** [comment]
> It will only exist if you run the game with the new version installed. Settings do not update until after the game is launched

Ah, that'd be it then. Was waiting for the latest offline updater from GOG at the time (I really don't like Galaxy; it's just GOG-Steam to me). Danke schoen!
